src/schemas/generator/v0/index.ts

```diff
@@ -30,18 +30,26 @@ export function migrate(generator: GeneratorSchema): v1.GeneratorSchema {
       },
     },
     testData: { ...generator.testData, files: [] },
-    // To please the schema addition defined in v1 we need to set the default value for extractionMode
-    // since the rule was working in single mode before the addition of multiple mode we define that as default
-    rules: generator.rules.map((rule) => {
+    rules: generator.rules.map((rule): v1.GeneratorSchema['rules'][number] => {
       if (rule.type === 'correlation') {
         return {
           ...rule,
           extractor: { ...rule.extractor, extractionMode: 'single' },
         }
       }
 
+      if (rule.type === 'verification') {
+        return {
+          ...rule,
+          target: 'status',
+          operator: 'equals',
+          value: { type: 'recordedValue' },
+        }
+      }
```

Comment:
Verification rule type wasn't set correctly (optional `selector`), but UI restricted creation of invalid rules, so we don't need a new migration.